tests/test_compute_jobs.py

```diff
@@ -13,27 +13,34 @@
 def test_good_response():
     with requests_mock.Mocker() as mock:
```

Comment:
This is duplicated across all the tests.  Use a `pytest.fixture`

Reply:
Will do Dr. Coleman. Thank you for the feedback. I noticed the others used the technique and their code looks much cleaner. I will work on cleaning this up next.